### PR TITLE
White Space on border sides problem solved

### PR DESCRIPTION
**Pull request by divyanshutewari17**
Issue:

### CODE REVIEW

**Review by dsnehasish74 — COMMENTED** [review]
Thank you great work . 
but my suggestion is can you make this fill process little smooth ? then it will be great

**Comment by divyanshutewari17** [comment]
What type of change in filling you want can you please elaborate?

**Comment by dsnehasish74** [comment]
Suddenly removing border on hover is looking little odd for me I guess . Can you please put that border as it is or remove that border after the color filled completely . It is my suggestion . Fell free to share your opinion.

**Comment by divyanshutewari17** [comment]
@dsnehasish74 I have changed that glitch button size is not changing when being hovered

**Review by dsnehasish74 — CHANGES_REQUESTED** [review]
It looks good . it solved the problem for basic and rounded button but for block button case the block buttons converted in to the basic button when hover on that can you please  fix that

**Comment by divyanshutewari17** [comment]
@dsnehasish74 I missed that sorry, now I have resolved that too

**Review by dsnehasish74 — APPROVED** [review]
Yah !! Now it's looking cool . Thank you for your work @divyanshutewari17 , Great. @shahednasser  now you can merge this pr ,I think.

**Comment by divyanshutewari17** [comment]
@dsnehasish74 Thank You.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your work @divyanshutewari17 but it seems that you made changes to `dist/sbuttons.css` directly? Please reflect the changes in the less files in the `src` directory and then run `npm run compile` and commit the changes. Don't manually change the `dist/sbuttons.css` file.

**Comment by divyanshutewari17** [comment]
@shahednasser change in .less file are not being reflected on the project

**Comment by divyanshutewari17** [comment]
@shahednasser I downloaded both the CSS files again and apply changes to the ".less" file. Now I guess its good to merge

File: dist/sbuttons.css [hunk 1]
```diff
@@ -2036,8 +2036,13 @@ a.yellow-btn:active:not(.fill-color-btn) {
...
 }
-.fill-color-btn:hover {
-  color: #fff !important;
+.fill-color-btn:not(.block-btn):hover{
+  width: 5.6rem;
+}
+.fill-color-btn:hover{
+  color: #fff  !important;
+  height: 3.62rem;
+  border: 0rem;
```
Comment (divyanshutewari17) [inline dist/sbuttons.css]: Changed

**Comment by shahednasser** [comment]
As I am not seeing the problem on my end, please @dsnehasish74 confirm that the problem is resolved

**Comment by dsnehasish74** [comment]
It's working fine  great

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @divyanshutewari17 for code and bug

**Pull request merged**